Sources/Logging/Logging.swift

```diff
@@ -96,6 +96,40 @@ public struct Logger {
     }
 }
 
+extension Logger {
+    @inlinable
+    func log(level: Logging.Level, message: @autoclosure () -> String, error: Error, file: StaticString = #file, function: StaticString = #function, line: UInt = #line) {
```

Comment:
I'm unsure about this one: should the `Error` be `@autoclosure` too?

Reply:
i was thinking about that too, but did not see when that would be useful, since its pretty binary? wdyt?

Reply:
I'd think either you have the error or not really, so leaving off the autoclosure should be fine for error. 

Since errors would most of the time be logged in any case I don't think it is worth saving for the case "if logging is turned off completely and the error is allocated on demand" which would be the only case that would benefit from making it lazy I think.

Good to keep as direct value IMO :)